### PR TITLE
Feature/filters

### PR DESCRIPTION
**Pull request by DIOLINK**
Captura de pantalla de 2021-03-17 00-17-08
Captura de pantalla de 2021-03-17 00-20-16
Captura de pantalla de 2021-03-17 00-19-28
Captura de pantalla de 2021-03-17 00-19-52
Peek 17-03-2021 00-23

### CODE REVIEW

File: src/components/Admin/UsersTable/index.js [hunk 1]
```diff
@@ -38,6 +99,10 @@ const UsersTable = () => {
     }
   };
 
+  const handleSelecteTitle = (title) => {
```
Comment (adrianponce89) [inline src/components/Admin/UsersTable/index.js:102]: handleSelectTitle

File: src/components/Admin/UsersTable/index.js [hunk 2]
```diff
@@ -1,24 +1,85 @@
...
+    'Acciones',
+  ];
 
   useEffect(() => {
     upDateTable();
   }, []);
 
+  const handleFilterUsers = (textSerch) => {
+    if (textSerch.length === 0) {
+      upDateTable();
```
Comment (adrianponce89) [inline src/components/Admin/UsersTable/index.js:52]: updateTable

File: src/components/Admin/UsersTable/index.js [hunk 3]
```diff
@@ -1,24 +1,85 @@
...
+    'Rol',
+    'Permisos',
+    'Acciones',
+  ];
 
   useEffect(() => {
     upDateTable();
   }, []);
 
+  const handleFilterUsers = (textSerch) => {
```
Comment (adrianponce89) [inline src/components/Admin/UsersTable/index.js:50]: textSearch

File: src/components/Admin/UsersTable/index.js [hunk 4]
```diff
@@ -1,24 +1,85 @@
...
+const MessageNotFound = styled.p`
+  font-size: 3em;
+`;
+
 const UsersTable = () => {
   const [selectedUsers, setSelectedUsers] = useState([]);
   const [users, setUsers] = useState([]);
   const [loading, setLoading] = useState(false);
+  const [filter, setFilter] = useState('');
+  const [serch, setSerch] = useState('Nombre');
```
Comment (adrianponce89) [inline src/components/Admin/UsersTable/index.js:34]: const [search, setSearch] = useState('Nombre');

**Review by adrianponce89 — CHANGES_REQUESTED** [review]

File: src/components/Admin/UsersTable/index.js (hunk 2 above)
Comment (DIOLINK) [reply src/components/Admin/UsersTable/index.js]: upDateTable to updateTable has been replaced in all tables

**Review by adrianponce89 — APPROVED** [review]
Awesome!

**Pull request merged**